armi/reactor/components/component.py

```diff
@@ -178,6 +178,13 @@ class Component(composites.Composite, metaclass=ComponentType):
         component types within the ARMI source code. All primitive shapes (such as a
         square, circle, holed hexagon, helix etc.) are derived from this base class.
 
+        Fundamental capabilities of this class include the ability to store arbitrary
+        parameters for each component in the ARMI model hierarchy and for the attributes
+        to define the properties of a physical piece of the reactor. The ``p`` attribute
+        contains the parameters of a Component, while the ``material``, and temperature
```

Comment:
>  The ``p`` attribute contains the parameters of a Component

while true, this isn't the whole picture. Parameters can be accessed like a dictionary. Accessing via c.p.<xyz> is a convenience via a fancy getter (@john-science could actually tell you how it works). And I don't think we need to include this information here, actually. Rather let's just keep the first sentence - I think that's adequate. But with a slight reword:

"Fundamental capabilities of this class include the ability to store parameters and attributes which describe the physical state of each Component within the ARMI data model."

Thoughts?